### DISCUSSION TITLE
✅ 🔗 Twitter X icon for Share Buttons widget (Duplicate of #23345)

### DISCUSSION THREAD

**Discussion by lisasettanta7** [elementor/elementor#25299] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Hi, I am writing this tiket to request help because in elementor's winget of social sharing there is no twitter-x logo and I would like to add it. is it possible ? thanks

### Describe the solution you'd like.

Hi, I am writing this tiket to request help because in elementor's winget of social sharing there is no twitter-x logo and I would like to add it. is it possible ? thanks

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by m1ga** [comment]
https://github.com/orgs/elementor/discussions/25200#discussioncomment-8596099

**Comment by nicholaszein** [comment]
Hey @lisasettanta7!

### As you might already know, the 𝕏 logo for Twitter was implemented with the release of Elementor and Elementor Pro version 3.20.0. 🙌 
You can now use the Icon Library to include the 𝕏 logo on any icon location on your website. 

### Social Icons
For the [Social Icons widget](https://elementor.com/help/social-icons-widget/), you can use the Icon Picker and find the 𝕏 icon in the icon library by typing either **_twitter_** or **`x `** (with a space) in the search box. 

**It's important to note that this icon is only available if you have <ins>purged Object Cache</ins>. Otherwise you won't see the icon in the library!**

### Share Buttons (Pro)
For the [Share Buttons widget (Pro)](https://elementor.com/help/share-buttons-widget-pro/), we used a different approach. To maintain Backward Compatibility we added a new Network called **X** to the Network Dropdown List. You need to manually select that option to see the new icon.

### Backward Compatibility
Elementor takes backward compatibility very serious. We have no intention to make changes to existing features that may break your site. For this reason we decided not to replace all icons with the new 𝕏 icon right away. This ensures all existing sites remain intact, and gives you the option to chose if you want to use the new icon or not.

----

Thank you everyone for your feedback and suggestions! 🙏 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
